src/App.jsx

```diff
@@ -32,16 +33,15 @@ function App() {
   }, []);
 
   useEffect(() => {
+    fetchStateData(currentStateId, setStateData, setError);
     fetchData("states.json", setStates, setError);
```

Comment:
I kept this in because we get the statesList for the state drop down fields from states.json. Without this file, we would have to define the full state list locally. However, this would mean we are loading this whole file anyway.
Let me know what you think. @rayestrada

Reply:
You are right about this. We can make a ticket to reduce the load of the states.json file. This is not a blocking issue for this PR.